### PR TITLE
Improve doc string formatting of denoise_bilateral

### PR DESCRIPTION
**Pull request by ahojnnes**

### CODE REVIEW

File: skimage/restoration/_denoise.py [hunk 1]
```diff
@@ -8,17 +8,18 @@
...
 
 def denoise_bilateral(image, win_size=None, sigma_color=None, sigma_spatial=1,
-                      bins=10000, mode='constant', cval=0, multichannel=True, sigma_range=None):
+                      bins=10000, mode='constant', cval=0, multichannel=True,
+                      sigma_range=None):
     """Denoise image using bilateral filter.
 
-    This is an edge-preserving and noise reducing denoising filter. It averages
-    pixels based on their spatial closeness and radiometric similarity.
+    This is an edge-preserving denoising filter. It averages pixels based on
```
Comment (JDWarner) [inline skimage/restoration/_denoise.py]: Comma after 'preserving'

**Comment by JDWarner** [comment]
One comment, otherwise :+1:

File: skimage/restoration/_denoise.py (hunk 1 above)
Comment (ahojnnes) [inline skimage/restoration/_denoise.py]: Right, thanks!

**Comment by soupault** [comment]
Thanks @ahojnnes !

**Pull request merged**